base_location_nuts/wizard/nuts_import.py

```diff
@@ -166,7 +166,7 @@ def create_or_update_nuts(self, node):
         if not self._check_node(node):
             return False
 
-        nuts_model = self.env['res.partner.nuts']
+        nuts_model = self.env['res.partner.nuts'].sudo()
```

Comment:
Why do you need `sudo`? The theory is that if you have access to the import menu, you should be able to import NUTS with your ACLs.